Hestia/Hestia/frontend/Local/UITableViewControllerServerDiscovery.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using UIKit;
+using Foundation;
+
+using Hestia.DevicesScreen.resources;
+using System.Drawing;
+using System.Collections;
+using Hestia.backend;
+using Hestia.backend.models;
+using Hestia.DevicesScreen;
+
+namespace Hestia
+{
+    public partial class UITableViewControllerServerDiscovery : UITableViewController
+    {
+		//AutoServerDiscovery autoServerDiscovery = new AutoServerDiscovery();
+
+		//NSMutableArray services = autoServerDiscovery.GetServices();

```

Comment:
Is someone from backend supposed to do this, or just work in progress?

Reply:
I think it is front end work. But I think you said that the Discovery was also still work in progress?

Reply:
Yes I have some methods to write to get the actual server interactor for the chosen server.